app/src/main/scala/org/alephium/explorer/service/FinalizerService.scala

```diff
@@ -0,0 +1,152 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.service
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import com.typesafe.scalalogging.StrictLogging
+import slick.basic.DatabaseConfig
+import slick.dbio.DBIOAction
+import slick.jdbc.PostgresProfile
+import slick.jdbc.PostgresProfile.api._
+
+import org.alephium.explorer.foldFutures
+import org.alephium.explorer.persistence._
+import org.alephium.explorer.persistence.model.AppState
+import org.alephium.explorer.persistence.schema.AppStateSchema
+import org.alephium.explorer.persistence.schema.CustomGetResult._
+import org.alephium.explorer.persistence.schema.CustomSetParameter._
+import org.alephium.util.{Duration, TimeStamp}
+
+/*
+ * Syncing mempool
+ */
+
+trait FinalizerService extends SyncService.BlockFlow
+
+object FinalizerService extends StrictLogging {
+
+  // scalastyle:off magic.number
+  val finalizationDuration: Duration = Duration.ofSecondsUnsafe(6500)
+  def finalizationTime: TimeStamp    = TimeStamp.now().minusUnsafe(finalizationDuration)
+  def rangeStep: Duration            = Duration.ofHoursUnsafe(24)
```

Comment:
How was the update performance after setting the timestamp range to be 24 hours?

Reply:
locally all good, on aws I'm still syncing the blocks, but so far so good

Reply:
Is it faster than 1600 update/s after increasing the range to 24 hours?

Reply:
I don't have much logs related to outputs update as I restarted the container with a new version after the all outputs update was done, but from the  few values I can find, it's still around this 1600/2000, it's not always constant as we don't have a constant number of inputs per 24h.

Reply:
okay